db-es-audit/src/find_s3.py

```
@@ -0,0 +1,153 @@
+    Exceptions:
+        General exception if the file cannot be read.
+    """
+
+    try:
+        response = s3_client.get_object(Bucket=AUDIT_BUCKET_NAME, Key=object_key)
+
+        # Read the content
+        object_content = response['Body'].read().decode('utf-8')
+        return object_content
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)